### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by abel12-pro**
Hey dear CR,
I created on this project,
mobile finish version which conains 
- [ ] works
- [ ] about me
- [ ] contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,5 +39,237 @@ <h3 class="let-connect">LET'S CONNECT</h3>
...
+                <p class="lang-text mt">Frameworks</p>
+                <img src="image/drop-down.svg" class="lang-drop-down mt" alt="">
+            </div>
+            <hr />
+            <div class="lang">
+                <p class="lang-text">Skills</p>
+                <img src="image/drop-down.svg" class="lang-drop-down" alt="">
+            </div>
+            <hr />
+        </div>
```
Comment (Alusp) [inline index.html:240]: - [ ] As part of the project requirement, please ensure to wrap all your `Technologies` in a `ul` tag and each of them in a `li` as part of the project requirement.
- [ ] And finally, Please ensure that your design matches the given design requirement, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention this design [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&mode=design&t=aPAqREn7zR7JAx5t-0):slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

 Student works

</td>
      <td>

Figma Design

</td>
    </tr>
  </table>

<table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

Student Button

</td>
      <td>

Figma button

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @abel12-pro, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descriptive ✔️
- [ ] Professional GitHub Flow used ✔️
- [ ] Profesional PR Summary ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @abel12-pro, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All changes fixed ✔️
- Linters free ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abel12-pro** [comment]
Thank you.

**Pull request merged**